src/Identity/IdentityServer/ResourceOwnerPasswordValidator.cs

```diff
@@ -113,7 +113,7 @@ public async Task ValidateAsync(ResourceOwnerPasswordValidationContext context)
             if (authRequest != null)
             {
                 var requestAge = DateTime.UtcNow - authRequest.CreationDate;
-                if (requestAge < TimeSpan.FromHours(1) && !authRequest.AuthenticationDate.HasValue &&
```

Comment:
I don't know much about this bit, but just curious if you have a better understanding.

Is there any issue with this date getting overwritten? Does this date have any uses/implication outside of passwordless that the overwriting of the date could interfere with?

Reply:
So that check is to ensure we don't validate passwordless requests that have stayed for too long and that if block only concerns passwordless.

Reply:
Understood, thank you!